### PR TITLE
feat(foss):[TRI-519] Change package structure to org.eclipse.tractusx…

### PR DESCRIPTION
**Pull request by ds-jkreutzfeld**
Separate pull request for the package move, otherwise git will get confused and think files were deleted / created when the other changes are applied directly.

### CODE REVIEW

File: irs-api/src/main/java/org/eclipse/tractusx/irs/bpdm/BpdmClient.java [hunk 1]
```diff
@@ -62,7 +61,7 @@ class BpdmClientImpl implements BpdmClient {
     private final RestTemplate restTemplate;
     private final String bpdmUrl;
 
-    /* package */ BpdmClientImpl(@Qualifier(OAUTH_REST_TEMPLATE) final RestTemplate restTemplate,
+    /* package */ BpdmClientImpl(@Qualifier(RestTemplateConfig.OAUTH_REST_TEMPLATE) final RestTemplate restTemplate,
```
Comment (ds-jhartmann) [inline irs-api/src/main/java/org/eclipse/tractusx/irs/bpdm/BpdmClient.java:64]: Static import got removed

File: irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java [hunk 2]
```diff
@@ -98,7 +98,7 @@ public JobOrchestrator(final TransferProcessManager<T, P> processManager, final
      * @param jobData additional data for the job to be managed by the {@link JobStore}.
      * @return response.
      */
-    public JobInitiateResponse startJob(final JobParameter jobData) {
+    public JobInitiateResponse startJob(final org.eclipse.tractusx.irs.dto.JobParameter jobData) {
```
Comment (ds-jhartmann) [inline irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java:101]: Something went wrong with the import here, I think

File: irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java [hunk 3]
```diff
@@ -241,7 +241,7 @@ private long startTransfers(final MultiTransferJob job, final Stream<T> dataRequ
 
     private TransferInitiateResponse startTransfer(final MultiTransferJob job,
             final T dataRequest)  /* throws JobErrorDetails */ {
-        final JobParameter jobData = job.getJobParameter();
+        final org.eclipse.tractusx.irs.dto.JobParameter jobData = job.getJobParameter();
```
Comment (ds-jhartmann) [inline irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java:244]: Something went wrong with the import here, I think

File: irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java [hunk 4]
```diff
@@ -254,7 +254,7 @@ private TransferInitiateResponse startTransfer(final MultiTransferJob job,
         return response;
     }
 
-    private Job createJob(final String globalAssetId, final JobParameter jobData) {
+    private Job createJob(final String globalAssetId, final org.eclipse.tractusx.irs.dto.JobParameter jobData) {
```
Comment (ds-jhartmann) [inline irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java:257]: Something went wrong with the import here, I think

File: irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java [hunk 5]
```diff
@@ -271,20 +271,20 @@ private Job createJob(final String globalAssetId, final JobParameter jobData) {
     }
 
     @SuppressWarnings("PMD.NullAssignment")
-    private net.catenax.irs.component.JobParameter buildJobParameter(final JobParameter jobData) {
-        return net.catenax.irs.component.JobParameter.builder()
-                                                     .depth(jobData.getTreeDepth())
-                                                     .direction(Direction.DOWNWARD)
-                                                     .aspects(jobData.getAspectTypes()
+    private JobParameter buildJobParameter(final org.eclipse.tractusx.irs.dto.JobParameter jobData) {
```
Comment (ds-jhartmann) [inline irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java:274]: Something went wrong with the import here, I think

File: irs-api/src/main/java/org/eclipse/tractusx/irs/semanticshub/SemanticsHubClient.java [hunk 6]
```diff
@@ -58,7 +57,7 @@ class SemanticsHubClientImpl implements SemanticsHubClient {
     private final RestTemplate restTemplate;
     private final String semanticsHubUrl;
 
-    /* package */ SemanticsHubClientImpl(@Qualifier(OAUTH_REST_TEMPLATE) final RestTemplate restTemplate,
+    /* package */ SemanticsHubClientImpl(@Qualifier(RestTemplateConfig.OAUTH_REST_TEMPLATE) final RestTemplate restTemplate,
```
Comment (ds-jhartmann) [inline irs-api/src/main/java/org/eclipse/tractusx/irs/semanticshub/SemanticsHubClient.java:60]: Static import got removed

File: irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/submodel/domain/SubmodelTestdataCreatorTest.java [hunk 7]
```diff
@@ -39,7 +40,7 @@ void shouldReturnAssemblyPartRelationshipWithPreDefinedChildrenWhenRequestingWit
                 catenaXId);
 
         final Set<ChildData> childParts = assemblyPartRelationship.getChildParts();
-        assertThat(childParts).hasSize(3);
+        Assertions.assertThat(childParts).hasSize(3);
```
Comment (ds-jhartmann) [inline irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/submodel/domain/SubmodelTestdataCreatorTest.java:43]: Static import got removed

File: irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/submodel/domain/SubmodelTestdataCreatorTest.java [hunk 8]
```diff
@@ -55,7 +56,7 @@ void shouldReturnAssemblyPartRelationshipWithCustomChildrenWhenRequestingWithCat
                 catenaXId, children);
 
         final Set<ChildData> childParts = assemblyPartRelationship.getChildParts();
-        assertThat(childParts).hasSize(3);
+        Assertions.assertThat(childParts).hasSize(3);
```
Comment (ds-jhartmann) [inline irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/submodel/domain/SubmodelTestdataCreatorTest.java:59]: Static import got removed

File: irs-api/src/test/java/org/eclipse/tractusx/irs/connector/job/InMemoryJobStoreTest.java [hunk 9]
```diff
@@ -273,7 +274,7 @@ void shouldFindCompletedJobsOlderThanFiveHours() {
         final List<MultiTransferJob> completedJobs = sut.findByStateAndCompletionDateOlderThan(JobState.COMPLETED,
                 nowPlusFiveHours);
         // Assert
-        assertThat(completedJobs).hasSize(1);
+        Assertions.assertThat(completedJobs).hasSize(1);
```
Comment (ds-jhartmann) [inline irs-api/src/test/java/org/eclipse/tractusx/irs/connector/job/InMemoryJobStoreTest.java:277]: Static import got removed on multiple occasions here

File: irs-api/src/test/java/org/eclipse/tractusx/irs/connector/job/PersistentJobStoreTest.java [hunk 10]
```diff
@@ -112,7 +113,7 @@ void addTransferProcess() {
         sut.addTransferProcess(job.getJobIdString(), processId1);
         refreshJob();
         assertThat(job.getTransferProcessIds()).containsExactly(processId1);
-        assertThat(job.getJob().getJobState()).isEqualTo(JobState.RUNNING);
+        Assertions.assertThat(job.getJob().getJobState()).isEqualTo(JobState.RUNNING);
```
Comment (ds-jhartmann) [inline irs-api/src/test/java/org/eclipse/tractusx/irs/connector/job/PersistentJobStoreTest.java:116]: Static import got removed on multiple occasions here

File: irs-api/src/test/java/org/eclipse/tractusx/irs/services/IrsItemGraphQueryServiceTest.java [hunk 11]
```diff
@@ -11,20 +11,19 @@
...
+import org.eclipse.tractusx.irs.component.Jobs;
+import org.eclipse.tractusx.irs.component.Relationship;
+import org.eclipse.tractusx.irs.component.enums.JobState;
+import org.eclipse.tractusx.irs.connector.job.JobStore;
+import org.eclipse.tractusx.irs.connector.job.MultiTransferJob;
+import org.eclipse.tractusx.irs.exceptions.EntityNotFoundException;
+import org.eclipse.tractusx.irs.persistence.BlobPersistence;
+import org.eclipse.tractusx.irs.persistence.BlobPersistenceException;
+import org.eclipse.tractusx.irs.util.JsonUtil;
+import org.eclipse.tractusx.irs.util.TestMother;
```
Comment (ds-jhartmann) [inline irs-api/src/test/java/org/eclipse/tractusx/irs/services/IrsItemGraphQueryServiceTest.java:26]: 1 import is missing here

File: irs-api/src/test/java/org/eclipse/tractusx/irs/util/TestMother.java [hunk 12]
```diff
@@ -105,7 +103,7 @@ public static MeterRegistryService simpleMeterRegistryService() {
     }
 
     public AASTransferProcess aasTransferProcess() {
-        return new AASTransferProcess(faker.lorem().characters(UUID_SIZE), faker.number().numberBetween(1, 100));
+        return new AASTransferProcess(faker.lorem().characters(IrsAppConstants.UUID_SIZE), faker.number().numberBetween(1, 100));
```
Comment (ds-jhartmann) [inline irs-api/src/test/java/org/eclipse/tractusx/irs/util/TestMother.java:106]: Static import got removed

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: ci/spotbugs-excludes.xml [hunk 13]
```diff
@@ -20,35 +20,35 @@
...
     <Match>
         <!-- This one is false-positive. -->
-        <Class name="net.catenax.irs.controllers.IrsExceptionHandler"/>
+        <Class name="controllers.org.eclipse.tractusx.irs.IrsExceptionHandler"/>
         <Method name="handleMethodArgumentTypeMismatchException"/>
         <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
     </Match>
     <Match>
-        <Class name="net.catenax.irs.util.JobMetrics$JobMetricsBuilder"/>
+        <Class name="util.org.eclipse.tractusx.irs.JobMetrics$JobMetricsBuilder"/>
```
Comment (ds-jhartmann) [inline ci/spotbugs-excludes.xml:51]: These package names don't seem quite right.

File: ci/spotbugs-excludes.xml [hunk 14]
```diff
@@ -20,35 +20,35 @@
...
         <!-- This one is false-positive. -->
-        <Class name="net.catenax.irs.controllers.IrsExceptionHandler"/>
+        <Class name="controllers.org.eclipse.tractusx.irs.IrsExceptionHandler"/>
         <Method name="handleMethodArgumentTypeMismatchException"/>
         <Bug pattern="BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
     </Match>
     <Match>
         <!-- This one is false-positive. -->
-        <Class name="net.catenax.irs.controllers.IrsExceptionHandler"/>
+        <Class name="controllers.org.eclipse.tractusx.irs.IrsExceptionHandler"/>
```
Comment (ds-jhartmann) [inline ci/spotbugs-excludes.xml:46]: These package names don't seem quite right.

File: ci/spotbugs-excludes.xml [hunk 15]
```diff
@@ -20,35 +20,35 @@
...
         <!-- We want to explicitly throw Exception in this method. -->
-        <Class name="net.catenax.irs.configuration.SecurityConfiguration"/>
+        <Class name="configuration.org.eclipse.tractusx.irs.SecurityConfiguration"/>
         <Method name="securityFilterChain"/>
         <Bug pattern="THROWS_METHOD_THROWS_CLAUSE_BASIC_EXCEPTION"/>
     </Match>
     <Match>
         <!-- This one is false-positive. -->
-        <Class name="net.catenax.irs.controllers.IrsExceptionHandler"/>
+        <Class name="controllers.org.eclipse.tractusx.irs.IrsExceptionHandler"/>
```
Comment (ds-jhartmann) [inline ci/spotbugs-excludes.xml:40]: These package names don't seem quite right.

File: ci/spotbugs-excludes.xml [hunk 16]
```diff
@@ -20,35 +20,35 @@
...
         <!-- CSRF protection turned off on purpose. -->
-        <Class name="net.catenax.irs.configuration.SecurityConfiguration"/>
+        <Class name="configuration.org.eclipse.tractusx.irs.SecurityConfiguration"/>
         <Method name="securityFilterChain"/>
         <Bug pattern="SPRING_CSRF_PROTECTION_DISABLED"/>
     </Match>
     <Match>
         <!-- We want to explicitly throw Exception in this method. -->
-        <Class name="net.catenax.irs.configuration.SecurityConfiguration"/>
+        <Class name="configuration.org.eclipse.tractusx.irs.SecurityConfiguration"/>
```
Comment (ds-jhartmann) [inline ci/spotbugs-excludes.xml:34]: These package names don't seem quite right.

File: ci/spotbugs-excludes.xml [hunk 17]
```diff
@@ -20,35 +20,35 @@
...
     </Match>
     <Match>
-        <Class name="net.catenax.irs.component.Shell$ShellBuilder"/>
+        <Class name="component.org.eclipse.tractusx.irs.Shell$ShellBuilder"/>
         <Bug pattern="UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR"/>
     </Match>
     <Match>
         <!-- CSRF protection turned off on purpose. -->
-        <Class name="net.catenax.irs.configuration.SecurityConfiguration"/>
+        <Class name="configuration.org.eclipse.tractusx.irs.SecurityConfiguration"/>
```
Comment (ds-jhartmann) [inline ci/spotbugs-excludes.xml:28]: These package names don't seem quite right.

File: ci/spotbugs-excludes.xml [hunk 18]
```diff
@@ -20,35 +20,35 @@
         <Bug pattern="EI_EXPOSE_REP2"/>
     </Match>
     <Match>
-        <Class name="net.catenax.irs.component.Shell$ShellBuilder"/>
+        <Class name="component.org.eclipse.tractusx.irs.Shell$ShellBuilder"/>
```
Comment (ds-jhartmann) [inline ci/spotbugs-excludes.xml:23]: These package names don't seem quite right.

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: ci/spotbugs-excludes.xml (hunk 18 above)
Comment (ds-jkreutzfeld) [reply ci/spotbugs-excludes.xml:23]: Already fixed in the restart branch.

File: ci/spotbugs-excludes.xml (hunk 17 above)
Comment (ds-jkreutzfeld) [reply ci/spotbugs-excludes.xml:28]: Already fixed in the restart branch.

File: ci/spotbugs-excludes.xml (hunk 16 above)
Comment (ds-jkreutzfeld) [reply ci/spotbugs-excludes.xml:34]: Already fixed in the restart branch.

File: ci/spotbugs-excludes.xml (hunk 15 above)
Comment (ds-jkreutzfeld) [reply ci/spotbugs-excludes.xml:40]: Already fixed in the restart branch.

File: ci/spotbugs-excludes.xml (hunk 14 above)
Comment (ds-jkreutzfeld) [reply ci/spotbugs-excludes.xml:46]: Already fixed in the restart branch.

File: ci/spotbugs-excludes.xml (hunk 13 above)
Comment (ds-jkreutzfeld) [reply ci/spotbugs-excludes.xml:51]: Already fixed in the restart branch.

File: irs-api/src/test/java/org/eclipse/tractusx/irs/util/TestMother.java (hunk 12 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/test/java/org/eclipse/tractusx/irs/util/TestMother.java:106]: Fixed on restart branch

File: irs-api/src/test/java/org/eclipse/tractusx/irs/services/IrsItemGraphQueryServiceTest.java (hunk 11 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/test/java/org/eclipse/tractusx/irs/services/IrsItemGraphQueryServiceTest.java:26]: Fixed on restart branch

File: irs-api/src/test/java/org/eclipse/tractusx/irs/connector/job/PersistentJobStoreTest.java (hunk 10 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/test/java/org/eclipse/tractusx/irs/connector/job/PersistentJobStoreTest.java:116]: Fixed on restart branch

File: irs-api/src/test/java/org/eclipse/tractusx/irs/connector/job/InMemoryJobStoreTest.java (hunk 9 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/test/java/org/eclipse/tractusx/irs/connector/job/InMemoryJobStoreTest.java:277]: Fixed on restart branch

File: irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/submodel/domain/SubmodelTestdataCreatorTest.java (hunk 8 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/submodel/domain/SubmodelTestdataCreatorTest.java:59]: Fixed on restart branch

File: irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java (hunk 5 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java:274]: IntelliJ switched up the imports, but it's still fine functionally.

File: irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java (hunk 4 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java:257]: IntelliJ switched up the imports, but it's still fine functionally.

File: irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java (hunk 3 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java:244]: IntelliJ switched up the imports, but it's still fine functionally.

File: irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java (hunk 2 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/JobOrchestrator.java:101]: IntelliJ switched up the imports, but it's still fine functionally.

File: irs-api/src/main/java/org/eclipse/tractusx/irs/bpdm/BpdmClient.java (hunk 1 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/main/java/org/eclipse/tractusx/irs/bpdm/BpdmClient.java:64]: Fixed in restart branch

**Review by ds-jhartmann — APPROVED** [review]

File: irs-api/src/main/java/org/eclipse/tractusx/irs/semanticshub/SemanticsHubClient.java (hunk 6 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/main/java/org/eclipse/tractusx/irs/semanticshub/SemanticsHubClient.java:60]: Fixed :)

File: irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/submodel/domain/SubmodelTestdataCreatorTest.java (hunk 7 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/submodel/domain/SubmodelTestdataCreatorTest.java:43]: Fixed

**Pull request merged**